src/app/frontend/common/components/labels/labels.html

```diff
@@ -14,6 +14,7 @@
 limitations under the License.
 -->
 
-<div class="kd-labels" ng-repeat="(key, value) in ::labelsCtrl.labels">
-  {{key}}: {{value}}
+<div flex="95" class="kd-labels" ng-repeat="(key, value) in ::labelsCtrl.labels">
```

Comment:
Labels have this rounded box around them. Text is actually smaller than this. Without this text will be so long that in some places the outer box will overlap other elements or be trimmed because it's hidden under them.